jmxtrans/pom.xml

```diff
@@ -354,8 +350,7 @@
 									fi</script>
 							</preremoveScriptlet>
 							<requires>
-								<require>java</require>
-								<require>/usr/bin/lsb_release</require>
+								<require>java >= 1.7</require>
```

Comment:
If you put a dependency on Java >= 1.7 in the packages, can you also update the configuration of the maven-compiler-plugin?